components/script/dom/webglrenderingcontext.rs

```diff
@@ -369,6 +451,13 @@ impl WebGLRenderingContext {
         true
     }
 
+    #[inline]
+    fn luminance(r: u8, g: u8, b: u8) -> u8 {
```

Comment:
Doc comments, at least describing a source for these constants (https://en.wikipedia.org/wiki/Relative_luminance may be enough)